cfpb/consumerfinance.gov
# Analytics: Migrate retirement analytics listeners off jquery

## Changes

- Analytics: Migrate retirement analytics listeners off jquery


## How to test this PR

1. Visit http://localhost:8000/consumer-tools/retirement/before-you-claim/?debug-gtm=true and click things and see that there are analytics events logged.
